Makefile

```diff
@@ -11,9 +11,15 @@ help:
 	@echo ''
 
 build:
+	python hooks/flat_schedule.py
+	python hooks/guidebook.py
+	python hooks/schedule_summary.py
```

Comment:
Why have these been added here?  Does wok now not invoke the hooks?

The problem with this is that these hooks get run *before* the site gets built.  Some of the hooks depend on the built HTML, so either they will work with old HTML, or will fail because there is not old HTML.